src/tritongrader/formatter.py

```diff
@@ -116,17 +116,17 @@ def basic_io_output(self, test: IOTestCase):
         if test.result.error:
             return "\n".join(
                 [
-                    "Unexpected runtime error!",
-                    "== stdout ==",
+                    "=== Unexpected autograder runtime error!  Please notify us on Piazza. ===",
```

Comment:
I think it's better say to "please notify your instructors" since not all classes use piazza (29 doesn't)